funnel/views/commentvote.py

```diff
@@ -88,40 +90,84 @@ def new_comment(self):
                 else:
                     flash(_("No such comment"), 'error')
             else:
-                comment = Comment(user=current_auth.user, commentset=self.obj.commentset,
-                    message=commentform.message.data)
+                comment = Comment(
+                    user=current_auth.user,
+                    commentset=self.obj.commentset,
+                    message=commentform.message.data,
+                )
                 if commentform.parent_id.data:
-                    parent = Comment.query.filter_by(suuid=commentform.parent_id.data).first_or_404()
-                    if parent.user.email:  # FIXME: https://github.com/hasgeek/funnel/pull/324#discussion_r241270403
-                        if parent.user == self.obj.user:  # check if parent comment & proposal owner are same
-                            if not current_auth.user == parent.user:  # check if parent comment is by proposal owner
-                                send_mail_info.append({'to': self.obj.user.email or self.obj.email,
-                                    'subject': u"{project} Funnel: {proposal}".format(project=self.obj.project.title, proposal=self.obj.title),
-                                    'template': 'proposal_comment_reply_email.md'})
+                    parent = Comment.query.filter_by(
+                        suuid=commentform.parent_id.data
+                    ).first_or_404()
+                    if (
+                        parent.user.email
+                    ):  # FIXME: https://github.com/hasgeek/funnel/pull/324#discussion_r241270403
+                        if (
+                            parent.user == self.obj.user
+                        ):  # check if parent comment & proposal owner are same
+                            if (
+                                not current_auth.user == parent.user
+                            ):  # check if parent comment is by proposal owner
```

Comment:
If we move the comments below the condition check, then we can have single lines for them.

Reply:
Actually, leave this. I'll fix it in the comment PR.